middleware/cors/cors_test.go

```diff
@@ -41,7 +42,13 @@ func Test_CORS_Wildcard(t *testing.T) {
 	handler := app.Handler()
 
 	// OPTIONS (preflight) response headers when AllowOrigins is *
-	app.Use(New(Config{AllowOrigins: "*", AllowCredentials: true, MaxAge: 3600}))
+	app.Use(New(Config{
```

Comment:
can you add an test with the default configuration ? and check then the expected headers/behavior